corehq/apps/accounting/migrations/0020_payment_method__unique_together.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.11 on 2018-03-26 13:46
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('accounting', '0019_standard_pricing_march_2018'),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
```

Comment:
How large is this table? These sort of migrations block writes to the table so if it takes a while on deploy it could be bad.

Would it be worth applying this manually to prod, where I assume it's the largest?

Reply:
This table is under 2000 rows on production, so I imagine it will be quick.  I've confirmed that the table is currently in a consistent state.